kube/services/netpolicy/networkpolicy_arborist_templ.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: networkpolicy-arborist
+spec:
+  podSelector:
+    matchLabels:
+      app: arborist
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: arborist
+      - podSelector:
+          matchLabels:
+             app: revproxy
+      - podSelector:
+          matchLabels:
+             app: arranger
+      ports:
+         - port: 80
+         - port: 443
+  egress:
+    - to:
+      - podSelector:
+          matchLabels:
+            app: arborist
+      - podSelector:
+          matchLabels:
+            app: fence
+      - ipBlock:
+           cidr: 0.0.0.0/0
+      - ipBlock:
+           cidr: 54.224.0.0/12
+      - ipBlock:
+           cidr: 52.92.16.0/20
```

Comment:
Does arborist need egress to the internet?

Reply:
oh I guess not yet